### DISCUSSION TITLE
How to configure the cross origins for a hybrid mobile app to access Keycloak? is it mandatory to enable localhost as allowed origins?

### DISCUSSION THREAD

**Discussion by anothergoodguy** [keycloak/keycloak#19218] [Q&A]
Hi,
 We are trying to trying to write a cordova mobile app that has to connect to our production keycloak instance. We followed the examples provided but both of the examples are expecting us to enable to allow origins for `localhost`. 

Could you please advise if there is an alternate way? we cant enable localhost as allowed origin.

thanks in advance!

**Comment by kiranchenna** [comment]
As per documentation you have to allow localhost..
I am also searching for alternative solution that I can block localhost. 

Thanks in advance

**Comment by anothergoodguy** [comment]
Hi All,

This is a blocker for us, any help on this will be a great value for us....
